### ISSUE TITLE
Shortcodes that print out images are not responsive.

### ISSUE THREAD

**Issue by navorite** [soflyy/oxygen-bugs-and-features#1726]
**Describe the bug**
Shortcode images are not responsive.

*A clear and concise description of what the bug is.*
If you insert a shortcode or shortcode wrapper that prints out an image (img tag), it will not be responsive (Will overflow from the page and create a very ugly page). However, If I add the image as an image element with the URL provided by the shortcode, it becomes responsive. 

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
Post link: https://tech-guidez.com/best-laptops-for-microsoft-office/

**To Reproduce**
1. Add a shortcode or shortcode wrapper that prints out a big image (On my example it was the AAWP plugin image shortcode ([amazon fields="B07YNK81F1"  value="thumb" image_size="large" image_align="center"]) for example.
2. Save the changes.
3. Go to your smartphone(preferably because it makes the page look strange) or any browser responsive mode.
4. Look at the shortcode, the image will not respond to the lower screens and will stay the same width.

**Expected behavior**
It is expected to resize itself to adapt to the lower sizes, just like how it would if I replace the shortcode with the image URL.

**What actually happens**
The image overflows at the sizes lower than the image width. And in my phone and Google's PageSpeed test, it creates extra width and zooms out the page.

**Screenshots**
[https://imgur.com/a/PvZ8BVr](url)

**Desktop (please complete the following information):**
- OS: Windows10
 - Browser: Firefox & Chrome
 - Oxygen Version 3.6

**Smartphone (please complete the following information):**
- Device: Samsung Galaxy S8 Plus + Other devices and tests such as Google PageSpeed 
 - OS: Android 
 - Browser Firefox & Chrome
 - Oxygen Version 3.6

**Comment by KittenCodes** [comment]
@navorite Please try setting the width and max-width on the Shortcode element to 100%. If that doesn't work, you will need to use CSS to prevent the image from being oversized.

**Comment by navorite** [comment]
> @navorite Please try setting the width and max-width on the Shortcode element to 100%.

I've tried doing that but it didn't fix it. The problem is that using Elementor, it works without any problem. 

> If that doesn't work, you will need to use CSS to prevent the image from being oversized.
Unfourtanetly, I don't know how to fix it with CSS :(

**Comment by navorite** [comment]
Actually, I found a fix! I found a plugin called WPDevDesign Oxygen Essentials and it fixed the image's responsiveness for me. And by checking the code, I actually found the piece of CSS that fixes it.

Here it is: `img {
  vertical-align: top;
  height: auto;
  max-width: 100%;
}`

A little disappointed that the plugin doesn't do it automatically for me. Added that comment for anyone who can't find a solution for this. Hope you guys add the fix in the following version.
